__tests__/api.test.js

```diff
@@ -93,7 +93,8 @@ describe('/api/articles/:article_id', () => {
                     created_at: expect.any(String),
                     article_img_url:
                       "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
-                      votes: 0
+                      votes: 0,
+                      comment_count: '2'
```

Comment:
We should create a new test and not modify the old one, which should be robust enough to not fail upon modification of the query :)